snowshu/core/models/relation.py

```diff
@@ -111,8 +111,8 @@ def relation(self, value: str) -> None:
     @staticmethod
     def quoted(val: str) -> str:
         """Returns quoted value if appropriate."""
-        return f'"{val}"' if all({val.isupper(),
-                                  val.islower(), }) and ' ' not in val else val
+        return f'"{val}"' if any({val.isupper(),
```

Comment:
looks like an issue in the original code

Reply:
Yeah, that first condition would never be able to evaluate as True.

However, we should remove this function entirely since we're moving the quoting functionality into the adapters.